### ISSUE TITLE
[BUG] Moonkin Aura stacks with 1/2 Elemental Oath, but not 2/2 Elemental Oath

### ISSUE THREAD

**Issue by raethkcj** [JamminL/wotlk-classic-bugs#512]
#### 1.1. Description
 Balance Druid's [Moonkin Aura](https://www.wowhead.com/wotlk/spell=24907/moonkin-aura) and Elemental Shaman's [Elemental Oath](https://www.wowhead.com/wotlk/spell=51466/elemental-oath) are both raid-wide spell crit auras.
 
 When 2/2 talent points are taken in Elemental Oath, the 5% crit aura does not stack with Balance Druid's Moonkin Aura.

However, when only 1/2 points are taken, the 3% crit aura *does*  stack with the 5% crit from Moonkin Aura, providing a total of 8% spell crit to the raid.
  #### 1.2. How to Reproduce
           1. Have a Shaman with 0/2 points in Elemental Oath, but enough unspent points to train 2/2
           2. Note the Shaman's spell critical strike chance
           3. Enter a party with a Balance Druid in Moonkin Form, note 5% increased spell crit
           4. Spend 1/2 talent points in Elemental Oath, note additional 3% spell crit
           5. Spend 2/2 talent points in Elemental Oath, note 3% reduction in spell crit
  #### 1.3  Source Material
Step 2:
![Neither](https://user-images.githubusercontent.com/7716908/199260149-ec218a8d-8ae2-4934-8681-93ec209d16db.jpg)
Step 3:
![Moonkin_0_Oath](https://user-images.githubusercontent.com/7716908/199260199-db8d0b48-cf73-467b-b531-ba017188fd62.jpg)
Step 4:
![Moonkin_1_Oath](https://user-images.githubusercontent.com/7716908/199260233-ec74faa2-a8f6-40c1-9afa-dfcd34c0da78.jpg)
Step 5:
![Moonkin_2_Oath](https://user-images.githubusercontent.com/7716908/199260317-264898cd-7492-43b5-b8f9-18a65d0eab4e.jpg)

#### 2.1. Description
  Moonkin Aura and Elemental Oath's spell critical strike chance should never stack
  ###
# 2.2 Source Material
N/A

**Comment by tacocat777** [comment]
It's worth mentioning, that it also works with two elemental shamans. One taking 1/2, the other taking 2/2. 

[And from what I was able to dig up from old '08 forum posts, this is most likely a old bug that was fixed in 3.0.2.](https://web.archive.org/web/20081218014900/http://forums.wow-europe.com/thread.html?topicId=5640524782&pageNo=1&sid=1) HOWEVER, Elemental Oath in it's current (3.3.5) form is far different from what it was in 3.0.2. 

When this change was implemented, there were 3 key differences from moonkin aura:
1) it was toggled, not passive
2) it required 2/2 ranks as opposed to 1/1
3) they added a unique/personal effect in 3.0.8 which increased spell damage done by the Shaman by 5/10% while Clearcasting is active

Meanwhile overtime they found ways to homogenize the "buff categories", however they did ultimately decide to keep the talent as a 0/2 option and give players more rewarding decision choices for their talents.

**I really think they should allow this buff specifically because as of 3.3.5, it is a unique to the elemental shaman class only. Allowing 1/2 Elemental Oath to stack with Moonkin aura gives raid leaders a reason to bring elemental shamans. It is also barely at a detriment to the shaman themselves as the added crit almost completely negates the dps loss from the missing 5% spell dmg done while clearcasting.
Elemental shamans have major scaling issues going into later phases, making them less appealing to most players in 2022 who want to min-max everything.**
